src/test/developTest/java/mServer/crawler/sender/wdr/WdrVideoUrlParserTest.java

```diff
@@ -0,0 +1,56 @@
+package mServer.crawler.sender.wdr;
+
+import java.util.Arrays;
+import java.util.Collection;
+import mServer.crawler.sender.newsearch.Qualities;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class WdrVideoUrlParserTest {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)